OneWelcomeExampleApp/ViewControllers/Profile/ProfileViewController.swift

```diff
@@ -33,4 +41,8 @@ class ProfileViewController: UIViewController {
     @IBAction func changePassword(_: Any) {
         profileViewToPresenterProtocol?.setupChangePinPresenter()
     }
+    
+    func setProfileName(_ profileName: String?) {
+        self.profileName = profileName
```

Comment:
the method should set the final value to the label.

Reply:
At the point when it is set the label does't exist yet that is why it is set to private property and then set to the label when available (viewDidLoad)

Reply:
because you use the method in the wrong context